recipes/r-alakazam/meta.yaml

```diff
@@ -0,0 +1,66 @@
+{% set version = '0.2.7' %}
+
+{% set posix = 'm2-' if win else '' %}
+{% set native = 'm2w64-' if win else '' %}
+
+package:
+  name: r-alakazam
+  version: {{ version|replace("-", "_") }}
+
+source:
+  fn: alakazam_{{ version }}.tar.gz
+  url:
+    - https://cran.r-project.org/src/contrib/alakazam_{{ version }}.tar.gz
+    - https://cran.r-project.org/src/contrib/Archive/alakazam/alakazam_{{ version }}.tar.gz
+  sha256: 83d781817163110e71d3f1ba2205284a33f5e8d6d654a7074e7e27a18e47fb85
+
+build:
+  number: 0
+
+  rpaths:
+    - lib/R/lib/
+    - lib/
+
+requirements:
+  build:
+    - r-base >=3.4.1
+    - r-rcpp >=0.12.11
+    - r-dplyr >=0.5.0
+    - r-ggplot2 >=2.0.0
+    - r-igraph >=1.0.0
+    - r-lazyeval
+    - r-progress
+    - r-readr
+    - r-scales
+    - r-seqinr
+    - r-stringi
+    - posix                # [win]
+    - {{native}}toolchain  # [win]
+    - gcc                  # [not win]
```

Comment:
Please remove the `# [win]` lines and add `gcc` and `llvm` as shown in https://bioconda.github.io/guidelines.html#c-c